packages/xdl/src/detach/installPackagesAsync.js

```diff
@@ -25,8 +25,12 @@ export default async function installPackagesAsync(
     const packageLockJsonExists: boolean = await fs.pathExists(
       path.join(projectDir, 'package-lock.json')
     );
-    const yarnExists = await commandExists('yarnpkg');
-    packageManager = yarnExists && !packageLockJsonExists ? 'yarn' : 'npm';
+    try {
+      await commandExists('yarnpkg');
```

Comment:
Looks like `command-exists` may return `false` in some cases and throw in some cases. We should not use that package. Let's use a similar check here as we do in `init`? https://github.com/expo/expo-cli/blob/bf16aeef6b72cfbe3f6b586013feeb524d0058cf/packages/expo-cli/src/commands/init.js#L163-L166

Reply:
Made a PR with this change: #341.